Src/AutoNSubstituteUnitTest/SubstituteAttributeTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Linq;
+using Xunit;
+using Xunit.Extensions;
+
+namespace Ploeh.AutoFixture.AutoNSubstitute.UnitTest
+{
+    public class SubstituteAttributeTest

```

Comment:
I'd be OK with deleting this test class. It tests no behaviour, and the type and its annotations are more concisely expressed by the class itself.